packages/@yodaos/application/application.js

```diff
@@ -75,6 +75,7 @@ function Application (options, api) {
 var ApplicationProto = {
   startService: startService,
   getService: getService,
+  getRuntime: getRuntime,
```

Comment:
IMHO runtime client should not necessarily be attached to Application instance. AFAIK the most related pattern would be `@yodaos/effect` and `@yodaos/keyboard`.

Reply:
Actually this is to expose the `setPhase()` and `idle()` to setup application, I don't think that's a good way for `@yodaos/application`, too.

@LanFly How about we have a class `SetupApplication` which encapsulates the related functions inside like `setPhase()`, `idle()` and `step()`?